galaxy_ng/tests/integration/utils/sync.py

```diff
@@ -0,0 +1,82 @@
+from ansible.errors import AnsibleError
+from ..utils import wait_for_task, cleanup_namespace, iterate_all
+
+
+def set_synclist(api_client, collections):
+    """Set the list of collections on the user's synclist and return the synclist object"""
+    r = api_client("_ui/v1/my-synclists/")
+    synclist_id = r["data"][0]["id"]
+    synclist = api_client(f"_ui/v1/my-synclists/{synclist_id}/")
+
+    synclist["collections"] = collections
+    api_client(f"_ui/v1/my-synclists/{synclist_id}/", method="PUT", args=synclist)
+
+    # the ansible-galaxy client doesn't know how to handle an empty response
+    try:
+        api_client(f"_ui/v1/my-synclists/{synclist_id}/curate/", method="POST")
+    except AnsibleError:
+        pass
```

Comment:
We have removed the curate task and the need to call that endpoint

The UI can stop calling this endpoint (that may already be done), then we can remove this
https://github.com/ansible/galaxy_ng/blob/da51d7ea1cced3007ea81a91c2862be4817bfa8b/galaxy_ng/app/api/ui/viewsets/my_synclist.py#L33-L40